web/src/App.js

```diff
@@ -1,23 +1,53 @@
-import React from "react";
-import logo from "./chef.png";
-import "./App.css";
+import React from 'react';
+import logo from './chef.png';
+import styled, { keyframes } from 'styled-components'
 
+const AppContainter = styled.div`
+  text-align: center;
+`
+const AppHeader = styled.header`
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+`
+const AppLogoSpin = keyframes`
```

Comment:
Ou, this is cool

Reply:
I know right! Gonna feel weird for a bit, and gotta minimize these to keep the editor clean, but I feel it'll be cool